script/i18n.coffee

```diff
@@ -15,10 +15,13 @@ module.exports =
 
     locale = 'en' if (!locale? or (!module.exports.strings and !module.exports.translations[locale]))
     string = if (!module.exports.strings) then module.exports.translations[locale][stringName] else module.exports.strings[stringName]
+    
+    clonedVars = _.clone(vars) or {};
```

Comment:
it's necessary to clone here or we could do in habitrpg-shared since it seems to be only needed for drops' notes?